sql-bom/build.gradle

```diff
@@ -9,6 +9,8 @@ micronautBom {
 }
 
 dependencies {
+    api platform("org.jdbi:jdbi3-bom:$jdbiVersion")
```

Comment:
Proper way of consuming a platform per https://docs.gradle.org/current/userguide/java_platform_plugin.html#sec:java_platform_consumption

Reply:
:+1: